backend/strapi-sync/create-multiple-item-campgrounds.js

```diff
@@ -517,8 +517,8 @@ async function createCampground(item) {
   );
 }
 
-async function createCampgrounds(items) {
-  await Promise.all(items.map(async (item) => createCampground(item)));
+export async function createMultipleItemsCampgrounds() {
+  await Promise.all(campgrounds.map(async (item) => createCampground(item)));
```

Comment:
Since `createCampground` just takes one parameter and it's the same one being passed by `.map`, we can simplify it by just passing `createCampground` itself as the callback

```suggestion
  await Promise.all(campgrounds.map(createCampground));
```